index.src.html

```diff
@@ -5,7 +5,8 @@ <h1>CORS and RFC1918</h1>
 ED: https://wicg.github.io/cors-rfc1918/
 Shortname: CORS-RFC1918
 Level: 1
-Editor: Mike West 56384, Google Inc., mkwst@google.com
+Editor: Titouan Rigoudy, Google, titouan@google.com
```

Comment:
Is 56384 your W3C member ID? I can find mine if that would be helpful.

Reply:
Yes. There's some metadata linking behind the scenes somewhere. If you have an ID, I'd be happy to put it in. If it's at all annoying to find, add it later?

Reply:
Not at all annoying, here it is: 124486.